packages/chatbot-server-mongodb-public/evalCases/uni_skills_evaluation_questions.yml

```
@@ -0,0 +1,188 @@
+- name: What is a skill on MongoDB?
+  messages:
+    - role: user
+      content: What is a skill on MongoDB?
+  reference: MongoDB offers skills, each skill should take between 60-90 minutes
+    to attain for a specific skill. You can learn skills across all topics in
+    MongoDB starting with Data Modeling. You can take an assessment which will
+    earn you a badge to indicate mastery of a specific skill.
```

Some wording issues on this one + I think this doesn't clearly define what a "skill" is. I think we can steal from mongodb.com/skills a bit here and frame around "skill badges".

```suggestion
  reference: MongoDB skills represent the knowledge and ability to work with MongoDB.
    This includes topics like data modeling, aggregation, performance, and more.
    MongoDB Skill Badges are free, focused credentials designed to help you quickly
    learn and validate specific MongoDB skills. Each skill badge takes roughly 60-90
    minutes to earn, including videos and hands-on labs followed by a short assessment
    to verify your mastery.
```